test/dlc_tests/ecs/conftest.py

```diff
@@ -2,21 +2,27 @@
 
 import pytest
 import boto3
+from botocore.exceptions import ClientError
 
 from test import test_utils
 import test.test_utils.ecs as ecs_utils
 
 
 @pytest.fixture(scope="session")
-def ecs_client():
-    return boto3.client("ecs")
+def ecs_client(region):
+    return boto3.client("ecs", region_name=region)
 
 
 @pytest.fixture(scope="function")
 def ecs_cluster_name(request):
     return request.param
 
 
+@pytest.fixture(scope="function")
+def ei_accelerator_type(request):
+    return request.param if hasattr(request, "param") else None
+
+
```

Comment:
This fixture is exactly the same as the one on `dlc_tests/conftest.py`. Will the tests not work without duplicating this fixture into `dlc_tests/ecs/conftest.py`?